commands/system/help.js

```diff
@@ -1,6 +1,8 @@
 const Command = require(`${process.cwd()}/base/Command.js`);
 const { MessageEmbed } = require("discord.js");
+const emojis = ["⏮", "◀", "⏹", "▶", "⏭", "🔢"];
```

Comment:
It's a constant that never changes -> The name could be uppercase.

Reply:
as in const Emojis?

Reply:
No, as in, `EMOJIS`. The common convention is to use all-uppercase letters.